#! /bin/sh /usr/share/dpatch/dpatch-run
## 08_grml_defaults.dpatch by Christian Hofstaedtler <ch@grml.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Pseudo-brand as grml

@DPATCH@
Index: b/scripts/live
===================================================================
--- a/scripts/live	2011-07-24 22:07:58.000000000 +0200
+++ b/scripts/live	2011-07-24 22:07:59.000000000 +0200
@@ -16,9 +16,9 @@
 root_snapshot_label="live-sn"
 home_snapshot_label="home-sn"
 
-USERNAME="user"
-USERFULLNAME="Live user"
-HOSTNAME="host"
+USERNAME="grml"
+USERFULLNAME="grml user"
+HOSTNAME="grml"
 
 mkdir -p "${mountpoint}"
 tried="/tmp/tried"
